src/Lsp/Uri.ml

```diff
@@ -0,0 +1,9 @@
+(* This file is part of DBL, released under MIT license.
+ * See LICENSE for details.
+ *)
+
+(** URI helper functions *)
+
+let to_path uri =
+  String.sub uri (String.length "file://")
+    (String.length uri - String.length "file://")
```

Comment:
Perhaps it should check that the prefix is indeed `"file://"`?

Reply:
I'll provide an update which uses the `uri` library to manage URIs (the current version doesn't work with percent-encoded strings), so this file will be gone. That said, I'm not sure what to do if someone provides a URI with a scheme different than `file` (e.g `http`). I don't think it's worth handling such cases.

Reply:
It seems that the only place where the path to the file is actually needed is when setting the directory to look for modules, so in the worst case it will just not find other modules. I think I'll leave it like that.